tasks/pages/_app.tsx

```diff
@@ -22,29 +21,45 @@ const spaceGrotesk = SpaceGrotesk({
 
 const queryClient = new QueryClient()
 
+const globalStyles = css`
+  @media (max-width: 1350px) {
+    .desktop-wrapper{
+      display: none;
+    }
+  }
+
+  @media (min-width: 1350px) {
+    .mobile-wrapper{
+      display: none;
+    }
+  }
```

Comment:
use ids instead of classes and parameterize the `1350px` into a variable.

Reply:
Variables cannot be used in combination with media queries https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries

Reply:
Can't you just move that variable to ts?